src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreWorker.cs

```diff
@@ -97,7 +98,7 @@ internal sealed class SolutionRestoreWorker : SolutionEventsListener, ISolutionR
             Lazy<Common.ILogger> logger,
             Lazy<INuGetErrorList> errorList,
             Lazy<IOutputConsoleProvider> outputConsoleProvider,
-            Lazy<INuGetExperimentationService> nuGetExperimentationService)
+            Lazy<INuGetFeatureFlagService> nuGetExperimentationService)
```

Comment:
nit-pick: Like Jeff's feedback. Can we modify variable names to represent feature flags instead of experimentation service?